mediator/src/controllers/encounter.ts

```diff
@@ -1,28 +1,25 @@
-import axios from 'axios';
-import { FHIR } from '../../config';
-import { logger } from '../../logger';
-import { generateFHIREncounterResource } from '../utils/encounter';
+import axios from "axios";
+import { FHIR } from "../../config";
+import { logger } from "../../logger";
+
 const { url: fhirUrl, username: fhirUsername, password: fhirPassword } = FHIR;
 
 export interface CHTEncounterDoc {
```

Comment:
Is the `CHTEncounterDoc` still necessary?

Reply:
No it isn't. I've removed it.